GameData/RealismOverhaul/Engine_Configs/AJ10_Adv_Config.cfg

```diff
@@ -76,28 +77,28 @@
 		CONFIG
 		{
 			name = AJ10-118F
-			minThrust = 42.3
-			maxThrust = 42.3
+			minThrust = 41.4
+			maxThrust = 41.4
 			heatProduction = 100
 
 			PROPELLANT
 			{
-				name = Aerozine50
-				ratio = 0.4654
+				name = UDMH
+				ratio = 0.406
 				DrawGauge = True
 			}
 
 			PROPELLANT
 			{
-				name = NTO
-				ratio = 0.5346
-                DrawGauge = False
+				name = IWFNA
+				ratio = 0.594
+                		DrawGauge = False
 			}
 
-			ullage = True
-			pressureFed = True
-			ignitions = 0
-
+			%ullage = True
```

Comment:
Minor nitpick: since nothing else is modifying the ModuleEngineConfigs at this pass (**FOR[RealismOverhaulEngines]**) the MM modifiers (**%**) are not needed, neither are the **!IGNITOR_RESOURCE,*{}** patches. ModuleEngineConfigs automatically overrides everything engine-related, including ignitions and any resources.

Since you are modifying this config file, could you also remove all other instances of these passes?

Reply:
Yes, but i was talking about just the ModuleEnginesConfigs node, not anything else outside it. The other things that you removed (part title/manufacturer/description and ModuleGenerator/RESOURCE) are required by the global engine configuration file.

Sorry if i was vague about the "nitpick" point!

Reply:
Should be fixed now

Reply:
Why did the numbers change on this? Do you have sources?

Reply:
Yes, the 138 has a Fuel-to-Oxidizer ratio of 2 [http://www.astronautix.com/a/aj10-138.html](source) and the 118K of 1.9 [http://www.astronautix.com/a/aj10-118k.html](source). The 118K's isp is 321 [https://engineering.purdue.edu/~propulsi/propulsion/rockets/liquids/aj10-118k.html](source1) [http://www.astronautix.com/a/aj10-118k.html](source2).

Reply:
I would not trust Astronautix for these. Space Launch Report (by **Ed Kyle**) is a much more credible source (although it does not mention MRs): http://spacelaunchreport.com/ELTT-Delta-Card.jpg

* The oxidizer type for the 118F variant are definitely wrong (the Isp looks like it is correct).
* The VAC Isp for the 118K variant seems correct. Various sources place it between 319 and 321, maybe 320 s would be a good compromise? The 1 s difference here and there can easily be lost in the noise.

Reply:
Changed!